Document IdentityCredentialError (#40986)
mdn/content: files/en-us/web/api/identitycredentialerror/error/index.md

---
title: "IdentityCredentialError: error property"
short-title: error
slug: Web/API/IdentityCredentialError/error
page-type: web-api-instance-property
browser-compat: api.IdentityCredentialError.error
---

{{APIRef("FedCM API")}}{{AvailableInWorkers}}

> The **`error`** read-only property of the {{domxref("IdentityCredentialError")}} interface

Reviewer: ...

## Value

> A known error from the [OAuth 2.0 specified error list](https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.2.1) or an arbitrary string.

Reviewer: ```suggestion
One of the values listed in the [OAuth 2.0 specified error list](https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.2.1) or an arbitrary string.
```
